src/model/strategies/data-extraction-strategy/src/DataExtractionStrategy.cpp

```diff
@@ -254,17 +254,18 @@ double DataExtractionStrategy::calculate_irradiation_gain(double azimuth_angle_r
 
 void DataExtractionStrategy::transform_detections_to_logical_detections(osi3::SensorData& sensor_data, const osi3::LidarDetectionData& detection_data) const
 {
-    osi3::Vector3d mounting_position;
-    osi3::Orientation3d mounting_orientation;
-    if (!profile.sensor_view_configuration.radar_sensor_view_configuration().empty())
+    MountingPosition mounting_pose;
+    if (sensor_data.sensor_view(0).radar_sensor_view_size() > 0)
```

Comment:
Wouldn't it be necessary to iterate over all SensorViews?

Reply:
Theoretically yes, I think currently in all the models on OpenMSL we assume, that there is only one SensorView coming in. We would need to restructure all the models, as we would need to find a mechanism that tells the model, which SensorView to use. In my opinion, this would be a separate issue.

Reply:
That's fine with me. Should we put this in an issue now or wait until we have the specific use case? At the moment, I suspect that the issue is not yet useful because we have not yet reached the point where this is being used in sensor simulation. What do you think?

Reply:
I just thought about it again. We always have only one SensorView, because we get one SensorView input in the model. It is then added to the SensorData object. So in this model, there can never be multiple SensorViews.

Reply:
Yes, right. But the question remains regarding RadarSensorView and LidarSensorView.

Reply:
You are right. Can you create an issue about the number technology specific sensor views? I agree, that we do not need to fix this right now, because there is currently no use-case for multiple sensor views. But with the issue it is documented, that we are aware of this.

Reply:
Done